### ISSUE TITLE
Support docsets autoupdate

### ISSUE THREAD

**Issue by JerryTheMouse** [zealdocs/zeal#307]
Not Updating the docsets frequently breaks them, so what about an auto-update feature ??
Suggestion for it : Make update as CLI command & we can use cron..

**Comment by trollixx** [comment]
Certainly, the auto update is definitely coming soon. But what do you mean by "breaks them". Docsets should update fine regardless of how many updates were skipped.

**Comment by JerryTheMouse** [comment]
Simply Zeal did not recognize all the docsets that exist in the docsets folder .. So I had to delete them from file-browser and re-download them ..
Note : That  I didn't update them for about 2 months

**Comment by trollixx** [comment]
The problem with not recognised docsets is coming from #310. A fix has been committed to 0.1 branch and will be available as part of 0.1.1 release shortly.

I'll leave this issue open as a feature request for docset auto-update.

**Comment by PhilippWendler** [comment]
A way to update all docsets from the command line would really be great.
